main.go

```diff
@@ -176,6 +187,49 @@ func buildArtifact(c config.Config) {
 	check(err, "Unable to move artifact.")
 }
 
+func buildFargateScheduledTaskArtifact(c config.Config) {
+	buildNumber := "unknown"
+	buildInfo, err := getBuildInfo(c)
+	if err == nil {
+		buildNumber = buildInfo.BuildNumber
+	}
+
+	makeDir(target, c.App)
+	makeDir(target, "cfn")
+
+	containerTag := fmt.Sprintf("%s.dkr.ecr.eu-west-1/%s:%s", c.AccountId, c.App, buildNumber)
+
+	buildOut, err := exec.Command("docker", "build", "-t", containerTag, ".").Output()
```

Comment:
What happens with the image once created - do we need to do a `docker push` here?